apps/wallet/src/features/Send/InputTransaction/TransactionFee/AbstractedTransactionFee.tsx

```diff
@@ -84,13 +86,21 @@ export const AbstractedTransactionFee: FC<Props> = ({ tokenList }) => {
 			};
 
 			setIsFeeLoading(true);
-			const fee = await requestTransactionFee(payload);
+
+			const { fee, requestID } = await requestTransactionFee(
+				payload,
+				feeRequestID,
+			);
+			if (requestID !== feeRequestID) return;
+			feeRequestID++;
```

Comment:
I prefer exposing fee value with metadata like mint address, then using the mint address  to ignore the mismatched updating